lib/Extension/ExpressionExtension.php

```diff
@@ -337,6 +338,12 @@ public function load(Container $container): void
                 new RealExpressionLanguage($container->get(Lexer::class), $container->get(Parser::class))
             );
         });
+        $container->register(ExpressionEvaluator::class, function (Container $container) {
```

Comment:
This could be used to simplify other classes which currently require both the EL and the evaluator.